### PR TITLE
Adding the SyncDocs tool + many French translations

### PR DESCRIPTION
**Pull request by iroelperez**

### CODE REVIEW

File: SyncDocs/SyncDocs.csproj [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <UserSecretsId>1630f78f-83d6-46aa-8b27-620733e6df2f</UserSecretsId>
```
Comment (ErikApption) [inline SyncDocs/SyncDocs.csproj:8]: Haven't looked at the details but any specific reason the project has a usersecretid?

File: SyncDocs/SyncDocs.sln [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.4.33205.214
```
Comment (ErikApption) [inline SyncDocs/SyncDocs.sln:4]: I'm thinking we should add the tool in our main datahub project for now - the idea is to keep this repo for the documentation only

**Review by ErikApption — CHANGES_REQUESTED** [review]
Can we move the tool to the main datahub solution?

File: SyncDocs/SyncDocs.csproj (hunk 1 above)
Comment (iroelperez) [reply SyncDocs/SyncDocs.csproj:8]: The DeepL secrect I stored locally.

**Review by ErikApption — APPROVED** [review]

**Pull request merged**